lib/utils/exit-handler.js

```diff
@@ -125,10 +121,7 @@ const exitHandler = err => {
 
   // only show the notification if it finished.
   if (typeof npm.updateNotification === 'string') {
-    const { level } = log
-    log.level = 'notice'
-    log.notice('', npm.updateNotification)
-    log.level = level
+    npm.forceLog('notice', '', npm.updateNotification)
```

Comment:
```
log.notice('', npm.updateNotification, { [log.META]: { force: true } })
```

In the future we could make `proc-log` able to do this and then even far aware log producers could do similar.

Reply:
Not a fan of the `[log.META]` but I do like the idea of having proc-log be where this intent is communicated. Let me think on it. Not for us to solve today.